tasks/main.yml

```diff
@@ -196,3 +196,19 @@
   notify:
     - Restart Jenkins with HTTPS
   when: jenkins2_https_enabled | bool
+
+- name: Apply Jenkins systemd configuration file and check HTTPS
+  template:
+    src: 'jenkins_system_config/jenkins.service'
+    dest: "{{ item }}"
+    owner: root
+    group: root
+    mode: 0644
+    backup: true
+  loop:
+    - /usr/lib/systemd/system/jenkins.service
+    - /lib/systemd/system/jenkins.service
```

Comment:
Why do we need two service files? If I'm not mistaken one is just link to another.

Reply:
It looks like location of jenkins.service depends on linux distribution.

After running 'apt install jenkins' on Ubuntu there are two equal jenkins.service files of the same date, size and content:
5633 Nov 30 10:32 /usr/lib/systemd/system/jenkins.service
5633 Nov 30 10:32 /lib/systemd/system/jenkins.service

However service jenkins.service runs from /lib/systemd/system/jenkins.service:
systemctl -l status jenkins
● jenkins.service - Jenkins Continuous Integration Server
     Loaded: loaded (/lib/systemd/system/jenkins.service; enabled; vendor preset: enabled)
     
On Amazon Linux 2 jenkins.service also exists in two locations (/usr/lib/systemd/system/jenkins.service and /lib/systemd/system/jenkins.service) but :
systemctl -l status jenkins
● jenkins.service - Jenkins Continuous Integration Server
   Loaded: loaded (/usr/lib/systemd/system/jenkins.service; disabled; vendor preset: disabled)
  (so /usr/lib/systemd/system/jenkins.service is actually used)

Reply:
Service files or unit files can be placed in any of the below paths

/etc/systemd/system/ - Prio 1
/run/systemd/system/ - Prio 2
/lib/systemd/system/ - Prio 3
/usr/lib/systemd/system/ - Prio 3

I think we should place customized service files always into `/etc/systemd/system/` because if you put them in `/usr/lib/systemd/system/` then they will be overwritten with default file when you install any updates.